db/re/item_db_equip.yml

```diff
@@ -143205,19 +143205,19 @@ Body:
     EquipLevelMin: 100
     Script: |
       bonus bAspdRate,10;
-      if (getpetinfo(PETINFO_ID) == 9109 || getpetinfo(PETINFO_ID) == 9112 || getpetinfo(PETINFO_ID) == 9115 || getpetinfo(PETINFO_ID) == 9121) {
+      if (getpetinfo(PETINFO_EGGID) == 9109 || getpetinfo(PETINFO_EGGID) == 9112 || getpetinfo(PETINFO_EGGID) == 9115 || getpetinfo(PETINFO_EGGID) == 9121) {
          bonus bBaseAtk,200;
          bonus bMatk,200;
          bonus bAllStats,10;
       }
       /*Todo Itemid 9126 9136*/
```

Comment:
The checks could already be added, even though they will never succeed - to get rid of the TODO.